osmaxx/excerptexport/models/export.py

```diff
@@ -164,7 +164,7 @@ def is_status_final(self):
 
     @property
     def can_be_deleted(self):
-        return self.is_status_final or self.update_overdue
+        return self.is_status_final or self.update_overdue()
```

Comment:
I think it would be a good idea, but I'm not happy with the name:
`update_is_overdue` or `is_update_overdue`?